### PR TITLE
Possible fix for #3735

### PR DESCRIPTION
**Pull request by redhatrises**
Possible fix for #3735. Really open to suggestions here as I can reproduce #3735 on RHEL7, but I can't consistently if at all reproduce this on Fedora.

### CODE REVIEW

**Comment by yuumasato** [comment]
>  but I can't consistently if at all reproduce this on Fedora.

This doesn't seem to be an issue in Fedora, I have cmake 3.12.1 installed and it works.
Although, documentation says wildcards are not supported: https://cmake.org/cmake/help/v3.12/manual/cmake.1.html#command-line-tool-mode

**Comment by redhatrises** [comment]
bump to anyone... need to know what we should do here.

**Comment by yuumasato** [comment]
After playing with cmake, I'm convinced that the problem is that cmake 2.8.12 doesn't support copying multiple files at once.
Globbing is not the problem, when a glob results in a single file, the copy command works.

I believe this PR is the best way to move forward and support cmake 2.8.12.

File: cmake/SSGCommon.cmake [hunk 1]
```diff
@@ -1107,7 +1107,7 @@ macro(ssg_build_zipfile ZIPNAME)
         COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_SOURCE_DIR}/Contributors.md" "zipfile/${ZIPNAME}"
         COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_SOURCE_DIR}/LICENSE" "zipfile/${ZIPNAME}"
         COMMAND ${CMAKE_COMMAND} -E make_directory "zipfile/${ZIPNAME}/kickstart"
-        COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_SOURCE_DIR}/rhel{6,7}/kickstart/*-ks.cfg" "zipfile/${ZIPNAME}/kickstart"
+        COMMAND ${CMAKE_COMMAND} -DFILES="${CMAKE_SOURCE_DIR}/rhel*/kickstart/*-ks.cfg" -DPATH="zipfile/${ZIPNAME}/kickstart" -P "${CMAKE_SOURCE_DIR}/cmake/CopyFiles.cmake"
```
Comment (yuumasato) [inline cmake/SSGCommon.cmake]: I suggest to follow same convention as `cp` command.
So rename `FILES` to `SOURCE`, and `PATH` to `DEST`.

**Comment by yuumasato** [comment]
I wish we could get rid of cmake 2.8.12....

**Comment by yuumasato** [comment]
@redhatrises I will take the liberty of pushing changes to this branch, :)

**Comment by yuumasato** [comment]
Rebase on latest master. Also:
- Extended use of `CopyFiles` cmake script
- Fixed zipfile target dependency on guides and tables
- Updated `nist-zipfile` target

**Comment by redhatrises** [comment]
> @redhatrises I will take the liberty of pushing changes to this branch, :)

Thanks @yuumasato! Didn't expect you to do that! much appreciated.

**Pull request merged**